framework/requirements/views/web/index.php

```diff
@@ -31,15 +31,15 @@ functionality may be not available in this case.
         </p>
 
         <h3>Conclusion</h3>
-        <?php if ($summary['errors'] > 0): ?>
+        <?php if ($summary['errors'] > 0) : ?>
```

Comment:
No PSR mentions it including PSR-12 in the works but Yii's internal doc does: https://github.com/yiisoft/yii2/blob/master/docs/internals/view-code-style.md

Accoding to the doc it should be:

```php
<?php if ($summary['errors'] > 0): ?>
```

i.e. no space.